Lib/tkinter/font.py

```diff
@@ -3,11 +3,12 @@
 # written by Fredrik Lundh, February 1998
 #
 
-__version__ = "0.9"
-
 import itertools
 import tkinter
 
+__version__ = "0.9"
+__all__ = ["NORMAL", "ROMAN", "BOLD", "ITALIC", "nametofont", "Font", "families",
```

Comment:
I would break a line after `"ITALIC"`.

Reply:
Sure thing. What is your rule for line breaks? I'd like to automate it in my IDE

Reply:
It is hard to automate. Semantically the first four names in this list are names of constants.